.github/workflows/nightly-ci.yml

```diff
@@ -0,0 +1,148 @@
+name: CI nightly
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Runs every day at 00:00
+    - cron: '0 * * * *' # Run every hour, for testing purpose
+
+permissions:
+  id-token: write
+  contents: write
+  packages: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+env:
+    REPORT_EXT: .junit-report.xml
+    ALLURE_REPORT_PATH: nightly-allure-report
+
+jobs:
+  ci_nightly:
+    uses: input-output-hk/catalyst-ci/.github/workflows/ci.yml@master
+    with:
+      aws_ecr_registry: 332405224602.dkr.ecr.eu-central-1.amazonaws.com
+      aws_role_arn: arn:aws:iam::332405224602:role/ci
+      aws_region: eu-central-1
+      nightly: true
+    secrets:
+      dockerhub_token: ${{ secrets.DOCKERHUB_TOKEN }}
+      dockerhub_username: ${{ secrets.DOCKERHUB_USERNAME }}
+      earthly_runner_address: ${{ secrets.EARTHLY_SATELLITE_ADDRESS }}
+      earthly_runner_secret: ${{ secrets.EARTHLY_RUNNER_SECRET }}
+
+  generate-test-reports:
+    name: Generate test reports
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup CI
+        uses: input-output-hk/catalyst-ci/actions/setup@master
+        with:
+            aws_role_arn: ${{ env.AWS_ROLE_ARN }}
+            aws_region: ${{ env.AWS_REGION }}
+            earthly_runner_secret: ${{ secrets.EARTHLY_RUNNER_SECRET }}
+
+      - name: Get catalyst gateway unit test report
+        uses: input-output-hk/catalyst-ci/actions/run@master
+        if: always()
+        continue-on-error: true
+        with:
+            earthfile: ./catalyst-gateway/
+            flags:
+            targets: build
+            target_flags:
+            runner_address: ${{ secrets.EARTHLY_SATELLITE_ADDRESS }}
+            artifact: "false"
+
+      - name: Get schemathesis test report
```

Comment:
You can remove this, the `Get flutter unit test report` and the `Get backend python test report` as we are interested only into the nighly targets in this workflow